Add E2E guide to run locally (#337)
millicast/millicast-sdk: CONTRIBUTING.md

It opens all the demo apps in your browser and keeps watching for changes in all the packages. You only need to refresh both pages if you modify the code.

### Running tests
If you want to run all tests, run:
```sh
npm run test
```

> > Note: There are some requirements to take into account before running E2E tests. Make sure to take a look at those.

Reviewer: Link to e2e testing section

#### Unit testing
If you want to run only unit tests, then you have to move to the `millicast-sdk` package:
```sh
cd packages/millicast-sdk
npm run test-unit
```

#### E2E testing
The first step before running E2E tests is setting some environment variables and adjusting the director URL desired.

> 1. You have to set three variables in the environment based on the token environment you are (production or develop). These variables are:

Reviewer: nitpick : best to have bash commands documented too. 

On mac, you can export these commands using

```bash
$ export PUBLISH_TOKEN=<your_publish_token>
$ export ACCOUNT_ID=<your_account_id>
$ export STREAM_NAME=<your_stream_name>
```

On a different note, can we not read them from a `.env` file?

Author: Will add this. Regarding the note, we are not reading from a `.env` file, but from the env vars exported to the terminal.

- `PUBLISH_TOKEN`
- `ACCOUNT_ID`
- `STREAM_NAME`

> 2. In `packages/millicast-sdk/src/config.js` set `MILLICAST_DIRECTOR_ENDPOINT` to the desired URL, either production (`https://director.millicast.com`) or develop (`https://director-dev.millicast.com`) environment.

Reviewer: We shouldn't be exposing the dev URL in a public/open source repository. We can remove the second point altogether because I don't believe the tests work against dev either.

It opens all the demo apps in your browser and keeps watching for changes in all the packages. You only need to refresh both pages if you modify the code.

### Running tests
If you want to run all tests, run:
```sh
npm run test
```

> Note: There are some requirements to take into account before running E2E tests. Make sure to take a look at [those](#e2e-testing).

#### Unit testing

> If you want to run only unit tests, then you have to move to the `millicast-sdk` package:

Reviewer: ```suggestion
If you want to run only unit tests, navigate to the `millicast-sdk` package. 
```
